tracer/src/Datadog.Trace/Propagators/DistributedContextExtractor.cs

```diff
@@ -63,6 +63,7 @@ private DistributedContextExtractor()
                           {
                               PropagatedTags = traceTags,
                               AdditionalW3CTraceState = w3CTraceState,
+                              IsRemote = true,
```

Comment:
This is a special extractor used for version-mismatch scenarios. Since each tracer version will have a different `SpanContext` type, we pass a `IReadOnlyDictionary<string, string>` between the automatic and manual tracers to share data like trace id, span id, etc.

I don't think a `SpanContext` that is crossing this boundary should be considered "remote" because all these spans are still part of the same "distributed" trace coming from the same service. The fact that the trace chunks are sent separately and connected later in the backend is an internal implementation detail.

But... it really depends on what we plan to use `IsRemote` for.

Reply:
Ahh thanks for that explanation. I agree it shouldn't be considered `IsRemote` for crossing the boundary, I'll remove it from `DistributedContextExtractor`.

Reply:
Thanks!